app/views/pages/supporters.html.erb

```diff
@@ -0,0 +1,56 @@
+<% content_for :page_title, t('.menu_title') %>
+<% content_for :wrapper_class, 'supporters-page' %>
+
+<div class="hero" id="masthead">
+  <div class="container">
+    <h1><%= t '.menu_title' %></h1>
+    <h2 class="lead"><%= t '.byline' %></h2>
+  </div>
+</div>
+
+<div class="page-header">
```

Comment:
This should probably be in the container, it feels weird to see it taking the whole page width.